.github/workflows/publish.yml

```diff
@@ -27,6 +27,7 @@ jobs:
           make -f Makefile.eve BRANCH?=${GITHUB_REF##*/} kernel-gcc
 
       - name: Log in to Docker Hub
+        if: github.ref == 'refs/heads/eve-kernel-amd64-v6.1.38-generic'
```

Comment:
won't 'docker push' fail? I guess we should disable both. Another question can we somehow define a branch name once e.g global variable so we do not have it all over yaml file? it would simplify merging a lot

Reply:
The if condition will disable this step in case of PR builds, in case of branch build the job will work as expected. Used the branch name as a variable